packages/web-components/fast-foundation/src/picker/picker.template.ts

```diff
@@ -0,0 +1,114 @@
+    const pickerMenutag: string = context.tagFor(PickerMenu);
+    const pickerListtag: string = context.tagFor(PickerList);
+    const defaultListItemTemplate: ViewTemplate = createDefaultListItemTemplate(context);
+    const defaultMenuOptionTemplate: ViewTemplate = createDefaultMenuOptionTemplate(
+        context
+    );
+    return html<Picker>`
+        <template
+            :selectedListTag="${() => pickerListtag}"
+            :pickerMenuTag="${() => pickerMenutag}"
+            :defaultListItemTemplate="${defaultListItemTemplate}"
+            :defaultMenuOptionTemplate="${defaultMenuOptionTemplate}"
+            @focusin="${(x, c) => x.handleFocusIn(c.event as FocusEvent)}"
+            @focusout="${(x, c) => x.handleFocusOut(c.event as FocusEvent)}"
+            @keydown="${(x, c) => x.handleKeyDown(c.event as KeyboardEvent)}"
+            @pickeriteminvoked="${(x, c) => x.handleItemInvoke(c.event as Event)}"
+            @pickeroptioninvoked="${(x, c) => x.handleOptionInvoke(c.event as Event)}"
+        >
+            <slot name="list-region"></slot>
+
+            ${when(
+                x => x.flyoutOpen,
+                html<Picker>`
+                <${anchoredRegionTag}
+                    class="region"
+                    auto-update-mode="auto"
+                    fixed-placement="true"
+                    vertical-positioning-mode="locktodefault"
+                    vertical-default-position="bottom"
+                    vertical-scaling="content"
+                    vertical-inset="false"
+                    horizontal-positioning-mode="locktodefault"
+                    horizontal-default-position="right"
+                    horizontal-scaling="anchor"
+                    horizontal-inset="true"
+                    @loaded="${(x, c) => x.handleRegionLoaded(c.event as Event)}"
+                    ${ref("region")}
+                >
+                    ${when(
+                        x => !x.showNoOptions && !x.showLoading,
+                        html<Picker>`
+                            <slot name="menu-region"></slot>
+                        `
+                    )}
+                    ${when(
+                        x => x.showNoOptions && !x.showLoading,
+                        html<Picker>`
+                            <div class="no-options-display" part="no-options-display">
+                                <slot name="no-options-region">
+                                    ${x => x.noSuggestionsText}
+                                </slot>
+                            </div>
+                        `
+                    )}
+                    ${when(
+                        x => x.showLoading,
+                        html<Picker>`
+                            <div class="loading-display" part="loading-display">
+                                <slot name="loading-region">
+                                    <fast-progress-ring
```

Comment:
Should this always use fast-progress?

Reply:
> I know this is just an example, but I like to see high contrast get some love in custom or implementation examples too :)

Done, keep the styling suggestions coming.

Reply:
fixed